src/sentry/static/sentry/app/components/charts/panelChart/index.jsx

```diff
@@ -0,0 +1,51 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import styled from 'react-emotion';
+
+import {Panel, PanelHeader} from 'app/components/panels';
+
+import Legend from './legend';
+
+const PanelChart = styled(
+  class extends React.Component {
```

Comment:
Do we still get PanelChart as the displayName?

Reply:
Updated https://github.com/getsentry/sentry/pull/9129/commits/b41b366af9a14da343a64d9b13f3eaddcb8e6867